protostar/commands/init/project_creator/new_project_creator.py

```diff
@@ -49,8 +51,15 @@ def _gather_input(self) -> "NewProjectCreator.NewProjectConfig":
 
         return NewProjectCreator.NewProjectConfig(project_dirname)
 
-    def _create_project(self, user_input: "NewProjectCreator.NewProjectConfig") -> None:
+    def _create_project(
+        self,
+        user_input: "NewProjectCreator.NewProjectConfig",
+        is_cairo_1: Optional[bool],
+    ) -> None:
         output_dir_path = self._output_dir_path
         project_root_path = output_dir_path / user_input.project_dirname
-        self.copy_template("default", project_root_path)
+        if is_cairo_1:
+            self.copy_template("cairo1", project_root_path)
+        else:
+            self.copy_template("cairo0", project_root_path)
```

Comment:
Plus, it is a bit confusing, in some places you use `is_cairo_1` as a `bool` flag, and in other places you use literals. Maybe it would be better to unify this?

I basically think we should create an enum with cairo options and use it here and in all other places where we conditionally use cairo0/cairo1, something like:
```
class CairoVersion(Enum):
    cairo0 = "cairo0"
    cairo1 = "cairo1"
```

Reply:
I think this one-enum-for-all thing can help a lot in the process of integrating with cairo1 but also deprecating cairo0 and all the things around because we'll be able to easily find all the spots where there are conditions regarding this. Otherwise, we're going to end up with a lot of flags with different types and names scattered around the codebase. I already suggested such an approach in https://github.com/software-mansion/protostar/pull/1522#discussion_r1116749852.

Reply:
Agree with the enum approach, will modify it today. 
Proposed change wouldn't work though, tried it already.

Reply:
Aren't those equivaltent 🤔 why it wouldnt work

Reply:
If you do the enum, the proposed change gets discarded anyhow but I also don't see what's wrong there, could you (out of curiosity) paste the error or something? Thanks

Reply:
<img width="630" alt="Screenshot 2023-03-02 at 13 54 53" src="https://user-images.githubusercontent.com/56825108/222434590-18db7b53-dc08-48f9-8812-92e2867b9108.png">

Reply:
What do you guys think about introducing the enum Karol proposed? 
I would put a file containing the enum in the main directory for visibility if we want to use it project-wide.
@MaksymilianDemitraszek @Arcticae

Reply:
As for the location, how about `protostar/cairo`?